src/EditorFeatures/Core/Implementation/IntelliSense/Completion/Controller_CommitUniqueCompletionListItem.cs

```diff
@@ -49,7 +49,7 @@ CommandState ICommandHandler<CommitUniqueCompletionListItemCommandArgs>.GetComma
             // we do want it through, it would be easy to get again simply by asking the model
             // computation to remove all filtering.
 
-            if (model.IsUnique)
+            if (model.IsUnique && model.SelectedItem != null)
```

Comment:
SelectedItem should be renamed to SelectedItemOpt if it can be null.  That will also help us audit all the locations where we use it to make sure they're ok.